pandas/io/formats/info.py

```diff
@@ -284,6 +425,17 @@ def memory_usage_bytes(self) -> int:
             deep = False
         return self.data.memory_usage(index=True, deep=deep).sum()
 
+    @doc(
+        INFO_DOCSTRING,
```

Comment:
can you keep INFO_DOCSTRING where it is and reference it here as `BaseInfo.render.__doc__`?

Reply:
I have just tried that, but it messes with the indentation for some reason

![image](https://user-images.githubusercontent.com/41443370/122974932-091b5d00-d3bd-11eb-838a-8c6d91cfdabe.png)